lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -34,7 +34,8 @@ def setup_rabbitmq_connection_config
         config :vhost, :validate => :string, :default => "/"
 
         # Enable or disable SSL
-        config :ssl, :validate => :boolean, :default => false
+        # Specify TLS version if using TLS e.g "TLSv1.2"
+        config :ssl, :validate => :string, :default => false
```

Comment:
Should this be a strict validation of `:validate => [ "TLSv1.2", "TLSv1.1", "TLSv1.0 "]`  only allowing those 3 values?